run

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# See: https://james-forbes.com/#!/posts/alternative-to-npm-scripts
+export PATH=./node_modules/.bin:$PATH
+
+function build(){
+    rm -fr dist || true
+    node scripts/build.cjs
+}
+
+function build-website(){
+    mkdir -p dist/website
+    cp website/index.html dist/website/index.html
+    cp -r website/assets/ dist/websites/assets
+}
+
+function ci(){
+
+    function secrets(){
+        curl --remote-name https://prerelease.keybase.io/keybase_amd64.deb
+        sudo apt install ./keybase_amd64.deb -y
+        git config --global user.email "deploys@vellajs.org"
+        git config --global user.name "Deploy Bot"
+        # You'll need the KEYBASE_PAPERKEY and KEYBASE_USERNAME variable set in order for this to work in CI
+        keybase oneshot -u "$KEYBASE_USERNAME" --paperkey "$KEYBASE_PAPERKEY"
+        rm -fr ./secrets || true
```

Comment:
Bash n00b... IIUC if one of the command fails, the rest of the function doesn't run, hence the `|| true`... But why would the `rm` fail here?

Reply:
That's basically there for local runs, the rm should always fail in ci, because `./secrets` won't exist yet, we clone on the next line.  But locally, secrets will likely still be there from the last run, so to guarantee the script behaves more closely to how it will in CI, we rm the directory first.